modular_skyrat/modules/gladiator/readme.md

```diff
@@ -0,0 +1 @@
+bob joga was here
```

Comment:
?

Reply:
bob wants code attribution I guess
he did recode the entire mob from scratch on a whim because he felt like it